presto-native-execution/scripts/dockerfiles/prestissimo-runtime.dockerfile

```diff
@@ -27,7 +27,8 @@ RUN mkdir -p /prestissimo /runtime-libraries
 COPY . /prestissimo/
 RUN EXTRA_CMAKE_FLAGS=${EXTRA_CMAKE_FLAGS} \
     make -j${NUM_THREADS} --directory="/prestissimo/" cmake-and-build BUILD_TYPE=${BUILD_TYPE} BUILD_DIR=${BUILD_DIR} BUILD_BASE_DIR=${BUILD_BASE_DIR}
-RUN ldd /prestissimo/${BUILD_BASE_DIR}/${BUILD_DIR}/presto_cpp/main/presto_server | awk 'NF == 4 { system("cp " $3 " /runtime-libraries") }'
+RUN LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/local/lib:/usr/local/lib64 ldd /prestissimo/${BUILD_BASE_DIR}/${BUILD_DIR}/presto_cpp/main/presto_server > /tmp/dynamic_libs.txt && ! grep "not found" /tmp/dynamic_libs.txt
```

Comment:
We can pipe to grep and check. No need to write to a file.
```
!(LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/local/lib:/usr/local/lib64 ldd /prestissimo/${BUILD_BASE_DIR}/${BUILD_DIR}/presto_cpp/main/presto_server  | grep "not found")
```
We should also combine both to avoid an extra RUN step.
I see some approaches here
```
RUN test -d /app/bin -a ... \
 && mkdir /app/control/bin
```
 
 https://stackoverflow.com/questions/53096984/how-to-stop-building-when-i-use-dockerfile-build-image